README.md

```diff
@@ -70,6 +70,10 @@ Options:
                     "jasmine", "qunit", "YUI Test", or "custom".
                     Defaults to "mocha".
                     See https://goo.gl/5KfjDS for details.
+                    
+  -T, --tunnel      Tunnel which will be created for running tests.
```

Comment:
Can you make this "-s, --service"? I'd rather not have two options with the same letter if possible.